tests/gcp/hooks/test_bigquery.py

```diff
@@ -56,21 +55,108 @@ def test_bigquery_client_creation(
         )
         self.assertEqual(mock_bigquery_connection.return_value, result)
 
+    @mock.patch(
+        'airflow.gcp.hooks.base.CloudBaseHook._get_credentials_and_project_id',
+        return_value=("CREDENTIALS", "PROJECT_ID",)
+    )
+    @mock.patch("airflow.gcp.hooks.base.CloudBaseHook.__init__")
+    def test_bigquery_bigquery_conn_id_deprecation_warning(
+        self, mock_base_hook_init, mock_get_creds_and_proj_id
+    ):
+        bigquery_conn_id = "bigquery conn id"
+        warning_message = "The bigquery_conn_id parameter has been deprecated. " \
+                          "You should pass the gcp_conn_id parameter."
+        with self.assertWarns(DeprecationWarning) as warn:
+            hook.BigQueryHook(bigquery_conn_id=bigquery_conn_id)
+            mock_base_hook_init.assert_called_once_with(delegate_to=None, gcp_conn_id='bigquery conn id')
+        self.assertEqual(warning_message, str(warn.warning))
 
-class TestPandasGbqCredentials(unittest.TestCase):
     @mock.patch(
         'airflow.gcp.hooks.base.CloudBaseHook._get_credentials_and_project_id',
         return_value=("CREDENTIALS", "PROJECT_ID",)
     )
-    @mock.patch('airflow.gcp.hooks.bigquery.read_gbq')
-    def test_credentials_provided(self, mock_read_gbq, mock_get_creds_and_proj_id):
-        self.instance = hook.BigQueryHook()  # pylint: disable=attribute-defined-outside-init
+    def test_bigquery_insert_rows_not_implemented(self, mock_get_creds_and_proj_id):
+        bq_hook = hook.BigQueryHook()
+        with self.assertRaises(NotImplementedError):
+            bq_hook.insert_rows(table="table", rows=[1, 2])
 
-        self.instance.get_pandas_df('select 1')
+    @mock.patch(
+        'airflow.gcp.hooks.base.CloudBaseHook._get_credentials_and_project_id',
+        return_value=("CREDENTIALS", "PROJECT_ID",)
+    )
+    @mock.patch("airflow.gcp.hooks.bigquery.BigQueryHook.get_service")
+    def test_bigquery_table_exists_true(self, mock_get_service, mock_get_creds_and_proj_id):
+        project_id = 'bq-project'
+        dataset_id = 'bq_dataset'
+        table_id = 'bq_table_view'
```

Comment:
Maybe it's worth to convert those variables to constants? This will reduce boilerplate code ;)

Reply:
I think it is good idea however I will address it in the another separate PR for the whole file.

Reply:
@nuclearpinguin WDYT about https://github.com/apache/airflow/pull/6869 ?

Reply:
@nuclearpinguin PR mentioned in my previous comment was merged, I've rebased and adjusted slightly and now I think it looks like you suggested :)